unit-tests/algo/depth-to-rgb-calibration/CMakeLists.txt

```diff
@@ -102,3 +91,67 @@ dl_algo_file( "${F9440687}/2/binFiles/xCoeff_P_5_5089x12_double_00.bin" "")
 dl_algo_file( "${F9440687}/2/binFiles/xCoeff_P_6_5089x12_double_00.bin" "")
 
 dl_algo_file( "${F9440687}/2/binFiles/camera_params" "")
```

Comment:
Shouldn't this file move to ac1x/?

Reply:
we need 2 of them: in ac1x and under binFiles